pages/Rails.md

```diff
@@ -93,6 +93,22 @@ Oj::Rails.set_decoder() method replaces that method with the Oj equivelant.
    are used as keys or if a other non-String objects such as Numerics are mixed
    with numbers as Strings.
 
-3. To verify Oj is being used turn on trace and then set the
-   `Tracer.display_c_call = true` to see calls to C extensions.
-   
+3. To verify Oj is being used, turn on C extension tracing.
+   Set `tracer = TracePoint.new(:c_call) do |tp| p [tp.lineno, tp.event, tp.defined_class, tp.method_id] end`
+   or, in older Rubies, set `Tracer.display_c_call = true`.
```

Comment:
oh, apparently this is in newer rubies too, just need to `require 'tracer'`. Missed that, oops :(

noticed in #459

Reply:
I think folks should be able to figure that much out. :-)

Reply:
Nice hook. I'll start using that too.